flang/lib/Lower/Intrinsics.cpp

```diff
@@ -733,6 +733,14 @@ mlir::Value IntrinsicLibrary::genFloor(mlir::Type resultType,
   return builder.createHere<fir::ConvertOp>(resultType, floor);
 }
 
+// IAND
+mlir::Value IntrinsicLibrary::genIAnd(mlir::Type resultType,
```

Comment:
These member functions need to be declared into the class they belong to and plugged into the dispatch table:
Here:
https://github.com/flang-compiler/f18-llvm-project/blob/7ddff247379e69cd901572f1485752bdff77d0f9/flang/lib/Lower/Intrinsics.cpp#L118
And here:
https://github.com/flang-compiler/f18-llvm-project/blob/7ddff247379e69cd901572f1485752bdff77d0f9/flang/lib/Lower/Intrinsics.cpp#L168

Otherwise, it will not compile/work

Reply:
Shoot I'm sorry, git was giving me issues with merging so I decided to copy and paste the code by hand but I forgot to copy over the function decorations. I re added them and made sure to compile the branch and run check-flang and now everything is working. Can this PR be re-opened or does a new one need to be made?

Reply:
Just go for a new one.